k8s.io/canary.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+alias kc="kubectl --context=utilicluster --namespace=k8s-io-canary"
+
+kc apply \
+    -f configmap-nginx.yaml \
+    -f configmap-www-get.yaml \
+    -f configmap-www-golang.yaml \
+    -f deployment.yaml \
+    -f service-canary.yaml
+
+sleep 5
```

Comment:
Is there a better way to wait for the deployment to complete? 5 seconds may be too fast.